### PR TITLE
Add test for components

### PR DESCRIPTION
**Pull request by kwekubright**
### Description:

- In this feature, my coding partner and I added tests to all components and logic files in the project ✅ 
- All tests passed ✅ 
image

### CODE REVIEW

File: src/components/__tests__/calculate.test.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import calculate from '../../logics/calculate';
```
Comment (FrederickMih) [inline src/components/__tests__/calculate.test.js:1]: - [ ] All functionalities and scenarios of the calculate module should be covered using jest. It's from the requirements hence you'll kindly consider the scenarios where **buttonName** `is Zero(0), NotZero(!0), dot(.), equal(=), plus/minus(+/-), `

**Review by FrederickMih — CHANGES_REQUESTED** [review]
Hi @kwekubright  and @MohammedElseddik , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



# To Highlights

- PR is professional :heavy_check_mark: :+1: 
- All Tests Suites are passing :heavy_check_mark: :heavy_check_mark: 

## Required Changes ♻️

- [ ] Hi @kwekubright, this is a collaborative project hence all the project contributors should be present in the README.md file, what do you think?. 
- [ ] Your app compiles with this issue when I ran it:
Screenshot from 2022-07-06 17-55-55
You'll require to fix it. 


_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by kwekubright** [comment]
Hello @FrederickMih , Thanks for your review. My partner and I are working on resolving the issue regarding the calculate.js testing.

Regarding the issue with the image below, however, my partner and I do not face this on our local. We are unable to reproduce the error faced at your end. 
image

**Review by prolajumokeoni — APPROVED** [review]
Hi @Team

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
## Highlights
- No linter errors ✅ 
- Git flow was used correctly 🚀 
- Good design 💯 
- Good Pr title and description 💪🏿 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by prolajumokeoni** [comment]
> Hello @FrederickMih , Thanks for your review. My partner and I are working on resolving the issue regarding the calculate.js testing.
> 
> Regarding the issue with the image below, however, my partner and I do not face this on our local. We are unable to reproduce the error faced at your end. image

Hi, I can currently see the same error but your project works perfectly, you can read more about the error [here](https://stackoverflow.com/questions/69733256/failed-to-load-config-airbnb-to-extend-from-gitlab-ci)👍🏿

**Pull request merged**